Resources/doc/reference/form_types.rst

```diff
@@ -561,6 +561,35 @@ example above:
 Other specific field configuration options are detailed in the related
 abstraction layer documentation.
 
+Adding a FormBuilderInterface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can add Symfony `FormBuilderInterface` instances to the `FormBuilder`. This allows you to
+re-use a model form type. When adding a field using a `FormBuilderInterface`, the type is guessed.
+
+.. code-block:: php
+
+    <?php
+    // src/AppBundle/Admin/Post.php
+
+    class Post extend AbstractAdmin
+    {
+        protected function configureFormFields(FormMapper $formMapper)
+        {
+            $builder = $formMapper->getFormBuilder()->getFormFactory()->createBuilder(PostType::class);
+
+            $formMapper
+                ->with('Post')
+                    ->add($builder->get('title'))
+                    ->add($builder->get('body'))
+                ->end()
+                ->with('Author')
+                    ->add($builder->get('author'))
+                ->end()
+            ;
+        }
+    }
+
```

Comment:
can you add the PostType as an example to the docs? maybe something like:

imagine you have this formtype:

.....phpcode

you can reuse it like this:

.....phpcode